internal/vm/vsock.go

```diff
@@ -140,7 +32,7 @@ func VSockDialConnector(timeout time.Duration, udsPath string, port uint32) IOCo
 			timeoutCtx, cancel := context.WithTimeout(procCtx, timeout)
 			defer cancel()
 
-			conn, err := VSockDial(timeoutCtx, logger, udsPath, port)
+			conn, err := vsock.Dial(timeoutCtx, logger, udsPath, port)
```

Comment:
Can you upgrade the SDK to the latest version and use DialContext here instead? It is needed to have https://github.com/firecracker-microvm/firecracker-go-sdk/pull/390.